src/__tests__/model.test.ts

```diff
@@ -282,7 +283,9 @@ describe('model', () => {
             filter: { foo: 'foo' },
           },
         },
-      ];
+      ] as const;
```

Comment:
Can you also still use the `bulkWrite()` with an inline argument which is not marked `as const`? Do we have such a test? Let's add one, if not.

Reply:
I added a test that restores the original `operations` declaration, as well as one that tests an inline `operations` definition (and preserved the `as const` test, added here)